src/librustc_borrowck/borrowck/mod.rs

```diff
@@ -1236,10 +1238,11 @@ impl<'a, 'tcx> BorrowckCtxt<'a, 'tcx> {
                         let let_span = self.tcx.hir.span(node_id);
                         let suggestion = suggest_ref_mut(self.tcx, let_span);
                         if let Some(replace_str) = suggestion {
-                            db.span_suggestion(
+                            db.span_suggestion_with_applicability(
                                 let_span,
                                 "use a mutable reference instead",
                                 replace_str,
+                                Applicability::Unspecified,
```

Comment:
I believe this can be machine applicable, but if there are multiple attempted uses of an immutable reference, I don't know how `rustfix` handles it, it might attempt fixing them multiple times :-/

Leave as `Unspecified` for now with a a rewording of the review comment for people editing this file in the future.